settings.gradle

```diff
@@ -1,2 +1,15 @@
-rootProject.name = 'mvc-auth-commons'
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        jcenter()
+        maven {
+            name "JCenter Gradle Plugins"
```

Comment:
Did you test if this custom maven repository is required?

Reply:
you are right; not needed so removed.